index.html

```diff
@@ -8,15 +8,23 @@
   </head>
 
   <body>
-    <div id="user-interface" class="centered">
-      <textarea id="input"></textarea>
-      <br />
-      separator: <input id="separator-input" />
-      <br />
-      <pre id="output" class="scrollable"></pre>
-    </div>
-
-    <!-- initialize your application -->
+          <div class="player">
+        <video class="player__video viewer" src="652333414.mp4"></video>
```

Comment:
Maybe the name of the video file need to be adjust. what I upload in our repo is videobunny.mp4 hihihi